inventory_management_system_api/schemas/item.py

```diff
@@ -53,6 +53,7 @@ class ItemPatchRequestSchema(ItemPostRequestSchema):
     catalogue_item_id: Optional[str] = Field(
         default=None, description="The ID of the corresponding catalogue item for this item"
     )
+    system_id: Optional[str] = Field(default=None, description="The ID of the system that the item belongs to")
```

Comment:
Can `default=None` be removed now?
```suggestion
    system_id: Optional[str] = Field(description="The ID of the system that the item belongs to")
```

Reply:
This one is patch so it should be optional.